versions/3.0.md

```diff
@@ -1050,6 +1051,36 @@ Response with no return value:
 description: object created
 ```
 
+#### <a name="callbackObject"></a>Callback Object
+
+A container for possible out-of band callbacks from an operation. A callback may be returned from an operation, calling back to the path specified in the operation object.
+
+##### Patterned Fields
+Field Pattern | Type | Description
+---|:---:|---
+<a name="responseName"></a>Callback name | [Callback Operation Object](#operationObject) <span>&#124;</span> [Operation Object](#operationObject) | An operation object used to define a callback payload structure
```

Comment:
Both links here go to the same target (the operation object), which doesn't match the example (which looks more like the top level Paths object, with some variable substitution mixed in in the path).